packages/coinstac-ui/app/render/components/results/displays/images.jsx

```diff
@@ -27,6 +43,24 @@ class Images extends Component {
       return string;
     }
 
+    /**
+     * slugify
+     * Converts human readable string to a slug
+     * @param  {string} text Input String to Convert
+     * @return {string}     Converted String to Slug
+     */
+
+     // ignore weird class-methods-use-this lint error
+     // eslint-disable-next-line
+    slugify(text) {
```

Comment:
[Lodash's `kebabCase`](https://lodash.com/docs/4.17.10#kebabCase) does exactly this.

Reply:
Thanks for the tip. Changed to Lodash.